app/controllers/lead_providers/your_schools_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module LeadProviders
+  class YourSchoolsController < ::LeadProviders::BaseController
+    before_action :set_lead_provider
+
+    def index
+      @cohorts ||= Cohort.where(start_year: [2021, 2022])
+      @selected_cohort = if params[:selected_cohort_id]
+                           Cohort.find(params[:selected_cohort_id])
+                         else
+                           Cohort.find_by(start_year: 2021)
```

Comment:
Perhaps the current year? Or the latest cohort? But not always 2021...

Reply:
I've had this hardcoded in some of my prs lately as well. I think we need to define current_cohort as a concept and provide a helper method for it that we can use. That way we only have one place to update this.

Reply:
I noticed you created a ticket for it, I definitely agree we need it 🙂

Reply:
I think the current year will work fine - we could also make it configurable - For now lets not overcomplicate it - current year should do the trick until the requirements will evolve

Reply:
Done